comit/src/network.rs

```diff
@@ -22,19 +22,17 @@ impl fmt::Display for DialInformation {
     }
 }
 
-/// All the data Alice learned from Bob during the communication phase of a
-/// herc20 <-> halight swap.
+/// Data Alice learned from Bob during the communication phase.
 #[derive(Clone, Copy, Debug)]
-pub struct WhatAliceLearnedFromBob {
-    pub redeem_ethereum_identity: identity::Ethereum,
-    pub refund_lightning_identity: identity::Lightning,
+pub struct WhatAliceLearnedFromBob<A, B> {
+    pub redeem_identity: A,
+    pub refund_identity: B,
```

Comment:
We could name these like this as well to be more clear:
```suggestion
    pub alpah_redeem_identity: A,
    pub beta_refund_identity: B,
```

Reply:
We had the same idea on our version of this haha.